File: src/main/java/com/kyiv_kug/stream/jv/Streams.java
package com.kyiv_kug.stream.jv;


import java.util.Arrays;
import java.util.List;
import java.util.stream.Collectors;

public class Streams {
    public static void main(String[] args) {
        List<Integer> strings = Arrays.asList(1,2,3,4);
        List<String> strings1 = strings.stream().map(s -> s + " ").collect(Collectors.toList());

        strings1.forEach(System.out::println);

    }
}

File: src/main/java/com/kyiv_kug/stream/kt/Streams.kt
package com.kyiv_kug.stream.kt

import java.util.*


fun main(args: Array<String>) {
    val ints = List(5){ it + 1 }

    val strings: List<String> = ints
            .filter { it > 0 }
            .map { "$it " }

    strings.forEach (::print)
}